src/aol-hb.js

```diff
@@ -27,4 +32,27 @@ globalContext.init = (bidRequestConfig, placementsConfigs) => {
   };
 };
 
+globalContext.queue.push = (cmd) => {
+  if (typeof cmd === 'function') {
+    try {
+      cmd.call();
```

Comment:
I think it makes more sense to call it `.call` because you are calling (executing the function) at this point in time. `.called` seems more like it would be a flag or a property.